local-tests/chainrunner/chain.py

```diff
@@ -28,39 +29,41 @@ def __getitem__(self, i):
     def __iter__(self):
         return iter(self.nodes)
 
-    def bootstrap(self, binary, accounts, **kwargs):
-        """Bootstrap the chain. `accounts` should be a list of strings.
+    def bootstrap(self, binary, validator_accounts, accounts=None, **kwargs):
```

Comment:
Where are other testcases which pass `accounts` ? Maybe we can rework them so they use `validator_accounts` instead. I understand now we support both

Reply:
in this aleph-node repo